packages/runtime/src/widgets/Form/__tests__/Checkbox.test.tsx

```diff
@@ -0,0 +1,117 @@
+/* eslint-disable react/no-children-prop */
+import "@testing-library/jest-dom";
+import { BrowserRouter } from "react-router-dom";
+import { fireEvent, render, screen, waitFor } from "@testing-library/react";
+import { ScreenContextProvider } from "@ensembleui/react-framework";
+import type { PropsWithChildren } from "react";
+import { Button, Form } from "../../index";
+
+const FormTestWrapper: React.FC<PropsWithChildren> = ({ children }) => {
+  return (
+    <BrowserRouter>
+      <ScreenContextProvider
+        screen={{
+          id: "formTest",
+          name: "formTest",
+          body: { name: "Column", properties: {} },
+        }}
+      >
+        {children}
+      </ScreenContextProvider>
+    </BrowserRouter>
+  );
+};
+
+describe("Checkbox Widget", () => {
+  test("initializes with a binding value", async () => {
+    const logSpy = jest.spyOn(console, "log");
+    render(
+      <>
+        <Form
+          children={[
+            {
+              name: "Checkbox",
+              properties: {
+                id: "checkbox",
+                label: "Check Box",
+                value: true,
+              },
+            },
+          ]}
+          id="form"
+        />
+        <Button
+          label="Check Value"
+          onTap={{
+            executeCode: "console.log(form.getValues())",
+          }}
+        />
+      </>,
+      {
+        wrapper: FormTestWrapper,
+      },
+    );
+
+    const getValueButton = screen.getByText("Check Value");
+    fireEvent.click(getValueButton);
+
+    await waitFor(() => {
+      expect(logSpy).toHaveBeenCalledWith(
+        expect.objectContaining({ checkbox: true }),
+      );
+    });
+  });
+
+  test("updates when binding changes value", async () => {
```

Comment:
Please rename this test to `updates when calling setValue` and add a new test that binds the value to a storage variable and change the storage variable. See the corresponding `Date` test.